lib/publiccloud/ec2.pm

```diff
@@ -169,6 +169,9 @@ sub img_proof {
 
 sub cleanup {
     my ($self) = @_;
+    select_console 'tunnel-console', await_console => 0;
```

Comment:
AFAIK, I wouldn't select a console in this class. The console depends on the test-module and here we use this tunnel and serial_terminal.
So from my side this break the logic!

Reply:
@cfconrad i made some changes, removing this particular addings from the ec2.pm. i tried to reuse the `select_host_console`. i will resolve this and feel free to comment to the new commits